### PR TITLE
Update link to CoffeeScript source in README.md

### PR DESCRIPTION
**Pull request by mattmenefee**
Update the link to the CoffeeScript source in README.md.

### CODE REVIEW

**Comment by mattmenefee** [comment]
However, when I then run
```ruby
context = ExecJS.compile(source)
```
I get the following error: `SyntaxError: Unexpected token 'export'`. Is that expected?

File: README.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ A longer example, demonstrating how to invoke the CoffeeScript compiler:
 ``` ruby
 require "execjs"
 require "open-uri"
-source = open("http://coffeescript.org/extras/coffee-script.js").read
+source = URI.open("https://coffeescript.org/browser-compiler-legacy/coffeescript.js").read
```
Comment (byroot) [inline README.md:35]: While we're at it, we shouldn't suggest using open-uri in 2024.

Comment (byroot) [reply README.md:35]:
```ruby
require "net/http"
source = Net::HTTP.get(URI("https://coffeescript.org/browser-compiler-legacy/coffeescript.js"))
```

Comment (mattmenefee) [reply README.md]: @byroot updated.

**Pull request merged**